app/views/posts/index.html.erb

```diff
@@ -1,2 +1,41 @@
 <h1>All Posts</h1>
+<div>
+  <div class="user-card flex">
+    <div class="user-image border">
+      <img src="<%= @user.photo %>" alt="<%= @user.name %>'s photo" width="100%" heigth="100%">
+    </div>
+    <div class="user-info border">
+      <h2><%= @user.name %></h2>
+      <p class="posts-number">Number of posts: <%= @user.posts.count %></p>
+    </div>
+  </div>
+</div>
+
+<div>
+  <ul>
+    <% @posts.each do |post| %>
+      <li>
+        <div class="user-card left">
+          <h3><%= link_to post.title, user_post_path(@user, post) %></h3>
+          <br>
+          <p><%= post.text %></p>
+          <div class="right">
+            <span class="span-post">Comments: <%= post.comments.count %></span>
+            <span class="span-post">Likes: <%= post.likes.count %></span>
+          </div>
+        </div>
+        <div class="user-card left">
+          <ul>
+            <h5>All the Comments:</h5>
+            <br>
+            <% post.comments.each do |comment| %>
+            <li><%= comment.author.name %>: <%= comment.text %></li>
+            <% end %>
+          </ul>
```

Comment:
- [ ] On the `Post Index` page, I will suggest you make use of the `five_most_recent_comments` custom method that was created in the previous milestone to show only the five recent comments when on the posts index page :+1: 
- Your Post Index Page
![image](https://github.com/ClaudiaRojasSoto/Blog_App/assets/102699958/ffffab7c-446e-42d1-bae8-b3494ceccc74)
- The given wireframe
![image](https://github.com/ClaudiaRojasSoto/Blog_App/assets/102699958/b29a47ec-a237-4d5c-9a5e-97d0aa488e00)

Reply:
Done. Thank you @NahnahAJ